From f3e52d89b20a8740d18cd9f1124f299b64f06b36 Mon Sep 17 00:00:00 2001
From: Lars Wendler <polynomial-c@gentoo.org>
Date: Fri, 8 Mar 2019 14:48:13 +0100
Subject: [PATCH] Don't call mandb

There are other man implementateions beside man-db so it is not even sure
that the "mandb" binary even exists on all unices. Other than that, usually
there's a cron job running "mandb" on a daily basis.
---
 src/cli/CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/src/cli/CMakeLists.txt b/src/cli/CMakeLists.txt
index e59a911a..c3f97a2c 100644
--- a/src/cli/CMakeLists.txt
+++ b/src/cli/CMakeLists.txt
@@ -94,5 +94,4 @@ endif()
 
 if(APPLE OR UNIX)
     install(FILES keepassxc-cli.1 DESTINATION ${CMAKE_INSTALL_MANDIR}/man1/)
-    execute_process(COMMAND mandb -q)
 endif()
-- 
2.21.0

